Escape Academy/escapeacademy.asl

```diff
@@ -2,63 +2,40 @@ state("Escape Academy") { }
 
 startup
 {
-	var bytes = File.ReadAllBytes(@"Components\LiveSplit.ASLHelper.bin");
-	var type = Assembly.Load(bytes).GetType("ASLHelper.Unity");
-	vars.Helper = Activator.CreateInstance(type, timer, this);
+	Assembly.Load(File.ReadAllBytes("Components/asl-help")).CreateInstance("Unity");
 
 	vars.Helper.AlertLoadless();
 }
 
-init 
+init
 {
 	print("Escape Academy detected.");
 
-	vars.Helper.TryOnLoad = (Func<dynamic, bool>)(mono =>
+	vars.Helper.TryLoad = (Func<dynamic, bool>)(mono =>
 	{
-		var mgm = mono.GetClass("Escape.Metagame.MetagameManager");
-		var slm = mono.GetClass("Escape.Metagame.SceneLevelManager");
+		var mgm = mono["Escape.Metagame.MetagameManager"];
+		vars.Helper["MGMState"] = mono.Make<int>(mgm, "_instance", "_currentMetaGameState");
+		vars.Helper["AsyncLoading"] = mono.Make<bool>(mgm, "_instance", "sceneLevelManager", "_asyncLoadingLevel");
 
-		vars.Helper["currentMGState"] = mgm.Make<int>("_instance", "_currentMetaGameState");
-		vars.Helper["_asyncLoadingLevel"] = mgm.Make<bool>("_instance", "sceneLevelManager", slm["_asyncLoadingLevel"]);
-
-		var lsm = mono.GetClass("Escape.Metagame.LevelSelect.LevelSelectManager");
-		vars.Helper["LSState"] = lsm.Make<int>("Instance", "_currentLevelSelectState");
+		var lsm = mono["Escape.Metagame.LevelSelect.LevelSelectManager"];
+		vars.Helper["LSState"] = mono.Make<int>(lsm, "Instance", "_currentLevelSelectState");
 
 		var rm = mono.GetClass("Escape.Rooms.RoomManager");

```

Comment:
Should this be:
```suggestion
		var rm = mono["Escape.Rooms.RoomManager"];
```

Reply:
Fixed in 18480731f5d3a0d49b13a99a8574552ae87de055.